packages/ui/src/eth/bridge.ts

```diff
@@ -0,0 +1,54 @@
+import { BigNumber, Contract } from "ethers";
+import type { Transaction } from "ethers";
+import type { ApproveOpts, Bridge, BridgeOpts } from "../domain/bridge";
+import TokenVault from "../constants/abi/TokenVault";
+
+class ETHBridge implements Bridge {
+  // ETH does not need to be approved for transacting
+  Approve(opts: ApproveOpts): Promise<Transaction> {
```

Comment:
What's the reason behind using capitalized names for the class methods?

Reply:
it adopts a Go pattern of `capital frst letter is an exported method, lower case is private` for readability